features/slayers/index.js

```diff
@@ -248,18 +287,26 @@ class Slayers extends Feature {
 		} else if (this.hideSummonsForLoot.getValue()) {
 			this.renderEntityEvent.unregister();
 		}
-		if (this.betterHideDeadEntity.getValue()) {
+
+		if (this.BoxAroundMiniboss.getValue() || this.betterHideDeadEntity.getValue()) {
 			World.getAllEntitiesOfType(net.minecraft.entity.item.EntityArmorStand).forEach(name => {
-				if (name.getName().removeFormatting().split(" ")[name.getName().removeFormatting().split(" ").length - 1] === "0❤" ||
-					(
-						name.getName().removeFormatting().split(" ")[name.getName().removeFormatting().split(" ").length - 1].split("/")[0] === "0"
-						&& name.getName().removeFormatting().includes("❤"))
-				) {
-					name.getEntity()[m.setAlwaysRenderNameTag](false)
+				let MobName = `${name.getName().removeFormatting().split(" ")[0]} ${name.getName().removeFormatting().split(" ")[1]}`
+				if (this.BoxAroundMiniboss.getValue() && !this.bossSpawnedMessage && this.Miniboss[this.lastSlayerType]?.has(MobName) && !this.minibossEntity.includes([new Entity(name.getEntity()), this.lastSlayerType])) {
```

Comment:
!this.minibossEntity.includes([new Entity(name.getEntity()), this.lastSlayerType])
^ will always be true because array.includes(array2) will always return false